curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/record-collection.english.md

```diff
@@ -30,59 +30,58 @@ You may refer back to <a href="learn/javascript-algorithms-and-data-structures/b
 
 ```yml
 tests:
+  - text: You should not change the <code>collection</code> object's initialization
+    testString: assert(code.match(/var collection = {\s*2548[:] {\s*album[:] "Slippery When Wet",\s*artist[:] "Bon Jovi",\s*tracks[:] \[\s*"Let It Rock",\s*"You Give Love a Bad Name"\s*\]\s*},\s*2468[:] {\s*album[:] "1999",\s*artist[:] "Prince",\s*tracks[:] \[\s*"1999",\s*"Little Red Corvette"\s*\]\s*},\s*1245[:] {\s*artist[:] "Robert Palmer",\s*tracks[:] \[ \]\s*},\s*5439[:] {\s*album[:] "ABBA Gold"\s*}\s*};/g));
```

Comment:
```suggestion
    testString: assert(code.match(/var collection = {\s*2548: {\s*album: "Slippery When Wet",\s*artist: "Bon Jovi",\s*tracks: \[\s*"Let It Rock",\s*"You Give Love a Bad Name"\s*\]\s*},\s*2468: {\s*album: "1999",\s*artist: "Prince",\s*tracks: \[\s*"1999",\s*"Little Red Corvette"\s*\]\s*},\s*1245: {\s*artist: "Robert Palmer",\s*tracks: \[ \]\s*},\s*5439: {\s*album: "ABBA Gold"\s*}\s*};/g));
```

Reply:
Are you sure? I added them in because it wasn't working for me without them.

Reply:
I am fairly sure.  Colon are not special characters that I am aware of.  Copy/paste what I posted above and see if it works with your latest changes.

Reply:
definitely getting an error in two ways...
- tried to save the file with these changes - then seed = error
- save a working file - npm run develop - change file to this + save = error
might just be me - but definitely some errors

Reply:
I know the issue.  You have to surround the testString with single quotes to escape the colon in the yml.  I have edited my suggestion above.

Reply:
Single quotes looks like it worked for me - added to the PR